app/backend-dotnet/Program.cs

```diff
@@ -0,0 +1,72 @@
+using Azure;
+using Azure.AI.OpenAI;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Search.Documents;
+using Azure.Storage.Blobs;
+using IMJohn.Services;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Configuration;
```

Comment:
Style preference nit, I prefer having a _GlobalUsings.cs_ file will these marked as `global`. I personally prefer not seeing any `using` declarations at all (just my opinion here).